test/new-e2e/system-probe/test/micro-vm-init.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 set -eEuo pipefail
 
+export DD_TEST_INSTALL_PATH_OVERRIDE="/opt/datadog-agent"
```

Comment:
Is this where it would install anyway?
It appears that nothing else in the test changed, which makes me think that not setting this would still work.